src/data/hero.ts

```diff
@@ -8,11 +20,38 @@ export const getHeroAbilities = (
   case "Captain Zane":
     switch (index) {
     case 0:
-      return [ "TODO" ];
+      return [ HASTE ];
     case 1:
-      return [ "TODO" ];
+      return [
```

Comment:
We won't want to lose first-band abilities. Do you intend to concatenate all of the abilities from all available bands and update the array every time the hero levels up? What about when external effects grant a card new abilities?

Reply:
Sounds like this will be addressed in getInstanceAbilities()

Reply:
Correct, this will be abstracted out by `getInstanceAbilities()`. An instance could be a unit, building, hero, or upgrade, all of which store abilities slightly differently. In the case of a hero, this will handle checking which bands should currently be in effect.

It also made more sense imo to have each band include only the new abilities for that band, which reduces duplication and should be less susceptible to weird errors / typos.